R/tbl_survfit_quantiles.R

```diff
@@ -0,0 +1,245 @@
+#' Survival Quantiles
+#'
+#' Create a gtsummary table with Kaplan-Meier estimated survival quantiles.
+#'
+#' @param data (`data.frame`)\cr
+#'   A data frame
+#' @param y (`string`)\cr
+#'   A string with the survival outcome, e.g. `survival::Surv(time, status)`.
+#'   The default value is `"Surv_CNSR()"`, which is a shortcut for
+#'   `survival::Surv(time = AVAL, event = 1 - CNSR, type = "right", origin = 0)`.
+#' @param by ([`tidy-select`][dplyr::dplyr_tidy_select])\cr
```

Comment:
It makes sense for continuity with the code, but I was wondering why not something that says it is a strata column you are referring? (docs are super clear anyway)

Reply:
I am not quite sure I follow 🤔